### DISCUSSION TITLE
Cookie Security

### DISCUSSION THREAD

**Discussion by LokiDieKatze** [orestbida/cookieconsent#267] [Q&A] [answered]
Few question about the security of cookies. 
What step allows the encryption of the data?
Is it possible to enable the httpOnly property etc. ?
What other security layer is offered by cookiconsent?

**Comment by tillsanders** [comment, marked as answer]
I believe there is no encryption baked into cookieconsent. That really depends on the cookies your application sets and wether it choses to encrypt them. Cookieconsent is only setting one cookie itself, which is the cc_cookie that contains the user settings. So I think you're looking in the wrong place. Or I misunderstood your question.

Regarding httpOnly – you cannot set those on the client. That is exactly what makes them (more) secure. More on that here: https://stackoverflow.com/questions/14691654/set-a-cookie-to-httponly-via-javascript
